libsolidity/formal/CHC.cpp

```diff
@@ -486,16 +445,17 @@ void CHC::endVisit(FunctionCall const& _funCall)
 	case FunctionType::Kind::BareCallCode:
 	case FunctionType::Kind::BareDelegateCall:
 	case FunctionType::Kind::Creation:
+		SMTEncoder::endVisit(_funCall);
+		unknownFunctionCall(_funCall);
+		break;
```

Comment:
@ekpyron This change is to not make pure/view functions such as the ones below erase knowledge. This was causing one of the tests to ignore a constant value.

Reply:
The change makes sense - but I don't see that in this commit - but I guess it's elsewhere, sound be fine in any case!

Reply:
Although I *am* curious - which test are you talking about :-)?

Reply:
True, forgot to push the fix to the test, it's in the commit now